Riot/Modules/Onboarding/AuthenticationCoordinator.swift

```diff
@@ -395,8 +409,7 @@ final class AuthenticationCoordinator: NSObject, AuthenticationCoordinatorProtoc
         case .dummy:
             MXLog.failure("[AuthenticationCoordinator] Attempting to perform the dummy stage.")
         case .other:
-            #warning("Show fallback")
-            MXLog.failure("[AuthenticationCoordinator] Attempting to perform an unsupported stage.")
```

Comment:
Could we leave the `MXLog.failure` here? It makes sense to me to show the fallback in this instance, but we should never actually reach this point as we should in theory be reverting to fallback immediately.